src/pymmcore_plus/core/_sequencing.py

```diff
@@ -277,6 +278,21 @@ def _nope(reason: str) -> tuple[bool, str] | bool:
     elif cur_length >= core.getExposureSequenceMaxLength(cam_dev):  # pragma: no cover
         return _nope(f"Camera {cam_dev!r} {max_len=} < {cur_length=}")
 
+    # SLM
+    try:
+        if e1.slm_image != e2.slm_image:
+            return _nope("SLMImage sequences are not supported yet.")
+    except ValueError:
+        if not np.array_equal(e1.slm_image, e2.slm_image):
+            return _nope("SLMImage sequences are not supported yet.")
+        # slm_dev = core.getSLMDevice()
```

Comment:
I'm find with the above for now.  you can uncomment this and we'll just leave it as unimplemented

Reply:
removed comments